inst/tutorials/non_normal_models.Rmd

```diff
@@ -0,0 +1,183 @@
+  }
+  return(data.frame(intercepts = intercepts, slopes = slopes))
+}
+```
+
+Let's put it to the test:
+
+```{r simulate_N_samples_test, exercise = T, exercise.eval = T}
+simulate_N_samples(10, 5)
+```
+
+> **NB: We should pause for a second here and realise that what we are simulating is the *imaginary scenario* in which the social scientist has access to many different samples from the population. In reality, *researchers only have access to a single sample*! However, it is a very useful fiction to imagine what would happen if we did have access to an infinity of samples in order to be able to assess the reliability of the sample at hand.**
+
+Now we are ready to compare how our estimates' precision changes as a function of the sample size! For this, we will:
+
+- 1. Simulate 5,000 different samples of size N
+- 2. Estimate a slope and an intercept coefficient for each of these 5,000 samples
+- 3. Visualize our 5,000 estimates in a histogram.
+
+Run the following cell to see the histograms:
+
+```{r simulate_N_samples_hist, exercise = T}
+# Pick a value for `size`, do not change N!
+sim <- simulate_N_samples(size = 10, N = 5000)
+
+#DO NOT CHANGE THESE LINES
+hist(sim$intercepts, main = "Intercept Estimates", xlab="", xlim = c(0, 10), col = "lightblue");
+
+hist(sim$slopes, main = "Slope Estimates", xlab="", xlim = c(0, 4), col = "lightblue")
+```
+
+Start with a sample size of 10, then increase the sample size to 100 and then to 1000. As you can see, the precision of our estimates increases with the sample size! This property of the OLS estimates is called **consistency** and is a crucial assumption in econometrics.
+
+A useful summary statistic here is the standard-deviation of the difference between the true value of the parameter and our estimates, that is to say by how much do our estimates typically vary. These are called the *standard errors* or the estimates. Intuitively speaking, an estimate that varies very much from one sample to the other should be interpreted with more carefulness (and less believed) than one that is very constistent.
+
+
+Let's plot how standard errors evolve with the sample size:
+
+```{r plot-stderr}
+n <- c(5, 10, 20, 30, 50, 75, 100, 200)
+se <- NULL
+for (i in n){
+  sim <- simulate_N_samples(size = i, N = 1000)
+  se <- append(se, sd(sim$slope))
+}
+plot(n, se, xlab = "Sample Size", ylab = "Standard Errors of Estimated Slope")
+lines(n, se)
+```
+
+
+
+In fact, it can be shown that the variance of the sampling distribution of estimated slopes $\hat\beta_1$ is given by:
+
+$$\sigma^2_{\hat\beta_1} = \frac{1}{n} \frac{var[(X_i - \mu_X)\varepsilon_i]}{[var(X_i)]^2}$$
+
+So the standard errors are 
+$$\sigma_{\hat\beta_1} = \sqrt{\frac{1}{n} \frac{var[(X_i - \mu_X)\varepsilon_i]}{[var(X_i)]^2}}$$
+$$\ = \frac{1}{\sqrt{n}} \frac{\sqrt{var[(X_i - \mu_X)\varepsilon_i]}}{var(X_i)}$$
+
+Plugging numbers from our examples, 
+$$\sigma_{\hat\beta_1} \approx \frac{1}{\sqrt{n}} \cdot\frac{3.45}{9} $$
+
```

Comment:
... so that after this, we can have another plot where we vary $sigma$2$, keeping N fixed.